eng/test-job.yml

```diff
@@ -139,18 +150,30 @@ jobs:
           helixSource: ci/dotnet/coreclr/$(Build.SourceBranch)
 
         ${{ if eq(parameters.readyToRun, false) }}:
-          helixType: ${{ format('test/functional/cli/pri{0}', parameters.priority) }}
+          helixType: 'test/functional/cli/'
```

Comment:
@RussKeldorph This would fix the links to mc.dot.net
 https://github.com/dotnet/core-eng/pull/5245#issuecomment-464225583